java/jsf/containerWithTests/Application/sampleWebApp.FunctionalTests/src/main/java/functional_tests/SampleFunctionalTest.java

```diff
@@ -50,6 +50,7 @@ public void testAssertTitleWithSelenium() throws InterruptedException, Assertion
                 startTimestamp = (new Date()).getTime();
                 if(startTimestamp > endTimestamp)
                 {
+                    System.out.println("##vso[task.logissue type=error;]Test testAssertTitleWithSelenium failed with error: " + e.toString());
                     throw e;

```

Comment:
prefer doing just "throw". This will ensure that the stack trace does not change. Otherwise the stack trace will point to this line.

Reply:
Just using throw doesn't seem to work with java